bindings/python/crocoddyl/core/integrator/euler.cpp

```diff
@@ -42,9 +42,9 @@ void exposeIntegratedActionEuler() {
                                                 const Eigen::Ref<const Eigen::VectorXd>&,
                                                 const Eigen::Ref<const Eigen::VectorXd>&)>(
           "calcDiff", &IntegratedActionModelEuler::calcDiff, bp::args("self", "data", "x", "u"),
-          "Compute the time-discrete derivatives of a differential action model.\n\n"
-          "It computes the time-discrete partial derivatives of a differential\n"
-          "action model. It assumes that calc has been run first.\n"
+          "Compute the time-discrete derivatives of the integrated action model.\n\n"
```

Comment:
This comment is wrong, it was alright as before.
The integrator convert continuous to discrete, the differential action model is continuous thing.

The integrated action takes a differential and then produces an action model.
When we say compute ... of a differential ..., it means that it takes the differential to produce an action model.

Reply:
> The integrator convert continuous to discrete, the differential action model is continuous thing.

This statement is misleading. The differential action model is continuous is true. But `x(t) = integral(t0,t )( f(x) )` is the continuous integration. If we had implemented this, calcDiff would still return the time derivatives. The integrator that we implement doesn't *convert* continuous to discrete. It is either continous, or discrete. 

The statement should be "Computes the derivatives of the integrated action model wrt state and control".

Reply:
done